blocks/library/paragraph/index.js

```diff
@@ -152,7 +152,10 @@ registerBlockType( 'core/paragraph', {
 					</PanelBody>
 				</InspectorControls>
 			),
-			<BlockAutocomplete key="editable" onReplace={ onReplace }>
+			<Autocomplete key="editable" completers={ [
+				blockAutocompleter( { onReplace } ),
+				userAutocompleter(),
+			] }>
```

Comment:
Nicee 👍

Reply:
Thinking the "patterns" we have right now in `Editable` could be written like this cc @iseulde